app/client-app/src/app/pages/Editor/components/PageList/PageHeader.js

```diff
@@ -20,7 +41,18 @@ const PageHeader = ({ page, order, dragHandleProps }) => {
           <Icon as={FaGripVertical} />
         </Flex>
         <LightHeading mx={2} size="sm">
-          Page {order}
+          <Flex align="center">
+            {order}
+            <Input
+              variant="flushed"
+              borderRadius={0}
+              fontSize="1.3rem"
+              value={name}
+              ml={2}
+              onChange={(value) => setName(value.target.value)}
```

Comment:
don't declare handler functions inline in props generally. it's usually worse for performance than doing it in the function body and then passing a reference.

Also, in this case, the handler should be an action (from [here](https://github.com/decsys/decsys/blob/810ace8e54993aa0f4d4858b6eacaf80594ffa0e/app/client-app/src/app/pages/Editor/actions/pageListActions.js)) as that already has access to state and helpers (like `id` and `mutate`) so you don't have to get them yourself. You can get the action then from the actions context using its hook, e.g. how we do [here](https://github.com/decsys/decsys/blob/810ace8e54993aa0f4d4858b6eacaf80594ffa0e/app/client-app/src/app/pages/Editor/components/PageList/PageActionButtons.js#L53)